File: Assignment_22/Program3.java
// Accept  the number from user and check whethter it is digit or not 

import java.util.Scanner;

class Demo
{
    public int iNo;

    public Demo(int no)
    {
        this.iNo = no;
    }

    public boolean CheckDigit()
    {
        boolean bFlag = false;

        if(iNo >= 0 && iNo <= 9)
        {
            bFlag = true;
        }
            
        return bFlag;  
    }
}

class Program3
{
    public static void main(String A[])
    {
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter your input");

        int no = sobj.nextInt();

        Demo dobj = new Demo(no);

        bRet = dobj.CheckDigit();

        if(bRet == true)
        {
            System.out.println("The Number is digit");
        }
        else 
        {
            System.out.println("The number is not digit");
        }
    }
}

File: Assignment 3/Program3.c
// Input : 24
// Output : 1 2 4 6 8 12

#include<stdio.h>
void DisplayEvenFactor(int iNo)
{
    int iCnt = 0;
    
    if(iNo < 0)
    {
        iNo = -iNo;
    }
 
    for(iCnt = 1 ; iCnt <= iNo; iCnt++)
    {
    if((iNo % iCnt == 0) && (iCnt % 2 == 0))
    {
            printf("%d ",iCnt);
    }
    }
}

int main()
{
    int iValue = 0;

    printf("Enter the number\n");
    scanf("%d",&iValue);

    DisplayEvenFactor(iValue);
    return 0;
}